### DISCUSSION TITLE
Help when using WCT datagrid.

### DISCUSSION THREAD

**Discussion by He1loC** [unoplatform/uno#12488] [Q&A]
I need to build an app that can also run on Win 7,so I choose skia.wpf.
I need to use the datagrid so I followed [](https://platform.uno/docs/articles/uno-community-toolkit.html?tabs=tabid-winui#task-1---add-windows-community-toolkit-to-uno-projects)
At first I tried both
Uno.CommunityToolkit.WinUI.UI.Controls 
and the one without Uno prefix seperately.
Neither of them could compile.
So I follow the tutorial and add conditional referrence.It compiled succeccessfully.
`<ItemGroup Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'">
  <PackageReference Include="CommunityToolkit.WinUI.UI.Controls" Version="7.1.2" />
</ItemGroup>
<ItemGroup Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) != 'windows'">
  <PackageReference Include="Uno.CommunityToolkit.WinUI.UI.Controls" Version="7.1.100" />
</ItemGroup>`

Then a error console pops up
![image](https://github.com/unoplatform/uno/assets/37819455/ff9cc44b-dde2-4c78-b10b-25479b285b75)
and the column headers can't be dragged to adjust like in UWP apps.
How do I fix this?

**Comment by jeromelaban** [comment]
Thanks for the report. This was an oversight when porting the DataGrid. We'll be publishing an update which fixes this issue but in the meantime, you can use the following, in your `App` constructor in `App.cs`:

```csharp
#if HAS_UNO
CommunityToolkit.WinUI.UI.Controls.DataGridFeatureConfiguation.EnableInvalidateMeasureInMeasureOverride = true;
#endif
